support/sdk-test/README.md

```diff
@@ -0,0 +1,47 @@
+# SDK Container Test
+This docker compose setup enables testing of an SDK container with the package repo.
+
+## Setup
+
+### Building an SDK Container
+
+You will need to build an sdk container that is compatible with your host. Do this by building for either
+
+* avocado-container-x86_64
+* avocado-container-arm64
+
+Once the container image is built, you can import the image from the build directory
+
+```bash
+docker import build-container-x86_64/build/tmp/deploy/images/avocado-container-x86_64/avocado-image-container-avocado-container-x86_64.rootfs.tar.bz2 avocadolinux/sdk:dev
+```
+
+### Building target package repos
+
+You will also need to build for a target to populate the package repos needed to start the package-repo container. The remainder of the document will assume the use of qemux86_64 as the target type.
+
+Refer to the main documentation for information on how to configure the system to build Avocado OS for a target machine type.
+
+## Testing
+
+You can start the package-repo container and get a bash prompt in the sdk container with the following docker compose command
+
+```bash
+docker compose -f support/sdk-test/docker-compose.yml run sdk-test /bin/bash
```

Comment:
What do I set `REPO_PATH` to when I run this `docker compose` command?

This environment variable is undefined when executing `docker-compose.yml`.

Here is the version of Docker I am running:

```
Docker version 26.1.3, build 26.1.3-0ubuntu1~24.04.1
```

I have to use `docker-compose` instead of `docker compose`.